internal/azure/azure.go

```diff
@@ -90,15 +90,15 @@ func (k KeyVaultSigner) Sign(_ io.Reader, digest []byte, opts crypto.SignerOpts)
 
 	signed, err := k.client.Sign(context.Background(), k.baseURL, k.keyName, k.keyVersion, keyvault.KeySignParameters{Algorithm: algo, Value: &payload})
 	if err != nil {
-		return nil, fmt.Errorf("failed to sign against azure: %w", err)
+		return nil, fmt.Errorf("azure: failed to sign against azure: %w", err)
 	}
 
 	res, err := base64.RawURLEncoding.DecodeString(*signed.Result)
 	if err != nil {
 		return nil, fmt.Errorf("failed to base64 recode result: `%s` %w", *signed.Result, err)
 	}
 
-	log.Infof("signed %d bytes with %s for %s key %s", len(digest), algo, k.keyType, k.keyName)
+	log.Infof("azure: signed %d bytes with %s for %s key %s", len(digest), algo, k.keyType, k.keyName)
```

Comment:
Info or debug? I think debug, but if there are reasons for Info that's fine.

Reply:
I think all of them should probably Debug or Error